doc/website/for-developers/usage-guide.md

```diff
@@ -81,3 +81,13 @@ count = 100
 ```
 
 When no config file is specified, a hard-coded version similar to [default config](../iceoryx_posh/etc/iceoryx/roudi_config_example.toml) will be used.
+
+# CaPro protocol
+
+<!-- @todo Add table and explain different terminologies used in AUTOSAR, ROS and DDS -->
+
+This service model comes from AUTOSAR. It is maybe not the best fit for typical publish/subscribe APIs
+but it allows us a matching to different technologies. The event can be compared to a topic in other
```

Comment:
Service, instance and event are not mentioned before in this file. We should introduce them somehow before explaining them.
I'm not sure, but maybe the ``overview.md`` is the better place for this explanation.

Reply:
Is it ok for you to clean this up with v1.0?

Reply:
My goal was not to really introduce Autosar specific terminology in a general iceoryx overview (I mention it briefly). In principle it does not matter how those strings are called to use them. Refactoring in 1.0 should adapt the names at least in the user facing API. Clean up with 1.0.

Reply:
Added to https://github.com/eclipse/iceoryx/issues/408#issue-758449131